src/main/java/frc/robot/hardware/phoenix6/pigeon/PigeonHandler.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.hardware.phoenix6.pigeon;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.interfaces.IAccelerometer;
+import frc.robot.hardware.interfaces.IGyro;
+import frc.robot.hardware.interfaces.IVibrationGyro;
+import frc.robot.hardware.phoenix6.Phoenix6Device;
+import frc.robot.hardware.phoenix6.Pigeon2Wrapper;
+
+public class PigeonHandler extends Phoenix6Device implements IGyro, IAccelerometer, IVibrationGyro {
+
+	private final Pigeon2Wrapper pigeon;
+	private final String logPath;
+	private final PigeonOffsets offsets;
+
+	public PigeonHandler(String logPath, Pigeon2Wrapper pigeon, PigeonOffsets offsets) {
```

Comment:
Consoder changing the name to pigeonWrapper
```suggestion
	public PigeonHandler(String logPath, Pigeon2Wrapper pigeonWrapper, PigeonOffsets offsets) {
```